src/ObjectGraph.es6.js

```diff
@@ -33,6 +33,7 @@ function ObjectGraph(opts) {
 
 ObjectGraph.prototype.init = function(opts) {
   this.q = new TaskQueue(opts);
+  this.instanceQueue = new TaskQueue();
```

Comment:
Let's add a `TODO` to make this into a plugin or extension. We can leave it here for now, though.